docker-compose.yml

```diff
@@ -2,16 +2,17 @@ version: "3.9"
 services:
   serge:
     restart: on-failure
-    build: 
+    build:
       context: .
       dockerfile: Dockerfile
     volumes:
       - datadb:/data/db
       - weights:/usr/src/app/weights/
-      - /etc/localtime:/etc/localtime:ro 
+      - ./api/weights:/usr/src/app/weights
```

Comment:
This is already included above on line 10

Reply:
This is responsible to make the folder /usr/src/app/weights reflect the files that are added to the /weights folder, if the user puts a model inside there, it will be detected by the condition

Reply:
That path is already used above:

```
- weights:/usr/src/app/weights
```

Reply:
The below line is just for making one folder reflect the other, and this ` - weights:/usr/src/app/weights` creates a volume only

Reply:
` - weights:/usr/src/app/weights` creates the volume weights 
` - ./api/weights:/usr/src/app/weights ` - make the content of the folder /usr/src/app/weights receive all the files from the /api/weights folder

Reply:
You can test it with and without the line I added, just to see the difference

Reply:
any update?